basement_bot/plugins/logger.py

```diff
@@ -11,16 +11,16 @@ class Logger(MatchPlugin):
     PLUGIN_NAME = __name__
 
     async def match(self, ctx, _):
-        if not ctx.channel.id in self.config.input_channels:
+        if not ctx.channel.id in self.config.channel_map.keys():

```

Comment:
I think `if not ctx.channel.id in self.config.channel_map` will work. Not sure how expensive `keys()` is

Reply:
Let me try that

Reply:
Got it. Works.